src/thread/x86_64/clone.s

```diff
@@ -40,6 +40,7 @@ __clone:
 	xor %ebp,%ebp
 	pop %rdi
 	pop %r9
+	and $-16,%rsp
```

Comment:
This shouldn't be needed.  Clone maintains 16-byte alignment if the stack is initially 16-byte aligned, so this can only cause problems if the stack is not already 16-byte aligned.  The x86-64 SysV ABI mandates 16-byte-aligned stacks, so anything that is leaving the stack unaligned is a bug.